### ISSUE TITLE
Amiga Floppy disk LED always on

### ISSUE THREAD

**Issue by ami-commo-fan** [indigolemon/EZ-FD#2]
I made one (rev 0.1) because gerbers seem not updated.
It works well, but the Floppy disk activity LED is always shining regardless if motor is spinning or not. Tested both on A500 and A1200.
Any clue?

**Comment by indigolemon** [comment]
I'm really sorry, I only use it in a towered setup where I rely on the light on the srive itself - I assume its the light on the Amiga itself that stays on?

**Comment by ami-commo-fan** [comment]
Yes, exactly, the Amiga drive LED stays on. I was reading elsewhere that the LED stays on when _mtr0 signal is active low.
Looking at the schematics I see the motor on is paired with pin4 (_inuse) which I am not sure why.
